rollup.config.js

```diff
@@ -33,8 +23,8 @@ export default {
     replace({
       __BASE_DOMAIN__: production ? '' : 'http://localhost:8000',
       __GA_TRACKING_ID__: process.env.GA_TRACKING_ID,
-      __BASE_SEARCH_DOMAIN__: SEARCH_DOMAIN,
-      __GLEAN_DICTIONARY_DOMAIN__: 'https://dictionary.telemetry.mozilla.org',
+      __GLEAN_DICTIONARY_DOMAIN__:
+        'https://deploy-preview-834--glean-dictionary-dev.netlify.app',
```

Comment:
Needs to be replaced once the relevant code in the glean dictionary lands (pending).